tests/docs/smoke-all/2023/03/01/4004.qmd

```diff
@@ -0,0 +1,39 @@
+---
+title: "issue 4004"
+format: 
+  docx:
+    fig-cap-location: top
```

Comment:
Should we be checking for specific things with

```yaml
_quarto:
  tests:
    docx:
      ensureDocxRegexMatches: ...
```

here? Unless I'm missing something, this test would pass on the current `main`, right? In that case, it wouldn't be exercising the fix.